src/docs/product/integrations/notification-incidents/msteams/index.mdx

```diff
@@ -28,50 +28,33 @@ Sentry owner, manager, or admin permissions are required to install this integra
 If you click "Add" instead of "Add to Team", you won't get the welcome message to finish installation.
 </Alert>
 
-3. You should see a welcome message in the "General" channel within a few seconds and click "Complete Setup".
+3. You should see a welcome message in the "General" channel within a few seconds. Click "Complete Setup".
 
    ![](msteams-welcome-message.png)
 
 The integration should be installed at this point.
 
-In the next section, we'll walk you through configuring your notification settings.
 
 ## Configure
 
-Use Microsoft Teams for [alerts](#alert-rules) regarding issues, environments, deployment, etc.
+Use Microsoft Teams for [alerts](/product/alerts/) regarding issues, environments, deployment, etc.
 
 ### Alert Rules
 
-1. Confirm your Teams installation is configured globally for your Sentry organization by navigating to **Organization Settings > Integrations**. Click "Configuration(s)" to go to the configuration details page.
+To set up an alert rule visit **Alerts > Create Alert**. From here you can route notifications to your Team's channels.
 
-   ![](msteams-directory-installed.png)
+Select Microsoft Teams as an [issue alert](/product/alerts/alert-types/#issue-alerts) action:
```

Comment:
We can take out the extra screenshots, and this doesn't even have to be formatted as a procedure, but right now, this is the only place we explain this config process step-by-step, so I think we should include all the actual information, even if it's a shortened version of it. I've made the same edit on all of these.

```suggestion
In [issue alerts](/product/alerts/alert-types/#issue-alerts), select "Send a Microsoft Teams notification" in the actions dropdown and then select your team and channel:
```